runtime/onert/backend/cpu/ops/SplitLayer.cc

```diff
@@ -51,20 +51,17 @@ void SplitLayer::splitFloat32()
     outputDimsPtr.push_back(&outputDims[i]);
   }
 
-  std::vector<float *> outputFloatPtrs;
+  std::vector<T *> outputFloatPtrs;
```

Comment:
I think **outputFloatPtrs** is inappropriate because This function is Generic function.
What about remove **Float** ?

Reply:
good observation, will rename it. Thanks :).